tests/Feature/AdminUsers/AdminUserCrudGeneratorWithCustomControllerNameTest.php

```diff
@@ -38,6 +39,7 @@ public function admin_user_controller_name_can_be_namespaced(): void
 use Brackets\AdminAuth\Services\ActivationService;
 use Brackets\AdminListing\Facades\AdminListing;
 use Exception;
+use Illuminate\Support\Facades\Auth;
```

Comment:
Why is this required?

Reply:
This facade is used for the impersonal login feature https://github.com/BRACKETS-by-TRIAD/admin-generator/blob/laravel-7/resources/views/templates/admin-user/controller.blade.php#L298